wpimath/src/test/native/cpp/MathUtilTest.cpp

```diff
@@ -65,6 +65,69 @@ TEST(MathUtilTest, ApplyDeadbandLargeMaxMagnitude) {
       frc::ApplyDeadband(100.0, 20.0, std::numeric_limits<double>::infinity()));
 }
 
+TEST(MathUtilTest, ApplyDeadband2dUnityScale) {
+  const Eigen::Vector2d zero(0.0, 0.0);
```

Comment:
You could just put this inline in the places that use it, to match Java. Otherwise, declare this as constexpr instead of const, and name it `kZero` because it's a compile-time constant.